lib/logo/nrk-logo-nrk-super-title-below.svg

```diff
@@ -1 +1 @@
-.707-.117 4 4 0 0 0-.286-.011 2.5 2.5 0 0 0-.503.046q-.2.039-.38.135a.6.6 0 0 0-.234.216.56.56 0 0 0-.076.292c-.002.1.024.2.076.286q.076.123.193.21.126.093.269.158.148.072.304.128.163.059.345.105.181.047.362.111t.357.123q.263.09.509.221.237.126.42.322c.123.13.22.283.287.45m5.065-3.62 1.333-.199-.012 6.107h-1.04l-.186-.759-.089.082a3.3 3.3 0 0 1-.602.431q-.313.175-.66.27a2.8 2.8 0 0 1-.743.093 2.3 2.3 0 0 1-1.116-.252 1.6 1.6 0 0 1-.696-.759q-.234-.507-.234-1.301V29.09l1.333-.199v3.667c-.008.26.036.52.128.764a.9.9 0 0 0 .398.462c.217.11.458.162.701.152q.221 0 .439-.041a3 3 0 0 0 .83-.286q.105-.055.216-.119zm7.31.158a2.5 2.5 0 0 0-1.392-.368q-.445.002-.877.117a3.8 3.8 0 0 0-.877.356q-.075.043-.152.092v-.553l-1.332.21v8.338h1.332v-2.624q.32.135.661.2.421.08.848.076a2.96 2.96 0 0 0 1.508-.385 2.76 2.76 0 0 0 1.058-1.121q.392-.735.392-1.787a3.7 3.7 0 0 0-.298-1.524 2.33 2.33 0 0 0-.871-1.027m-.421 3.876a1.53 1.53 0 0 1-.655.713c-.325.168-.687.25-1.052.239q-.252 0-.503-.035a3 3 0 0 1-.514-.123l-.152-.051v-3.621l.011-.005a6 6 0 0 1 .462-.187 3 3 0 0 1 .415-.111q.192-.035.386-.035a2.1 2.1 0 0 1 1.011.222 1.4 1.4 0 0 1 .608.677c.148.367.218.76.205 1.156.012.399-.064.795-.222 1.161m6.672-.863c.23-.068.447-.179.637-.327.162-.13.292-.296.38-.485.086-.19.13-.398.128-.607a1.7 1.7 0 0 0-.158-.712 1.8 1.8 0 0 0-.467-.625 2.2 2.2 0 0 0-.778-.432 3.4 3.4 0 0 0-1.087-.157q-.96 0-1.625.402a2.54 2.54 0 0 0-1.006 1.133 4 4 0 0 0-.339 1.699q0 .969.357 1.635c.238.444.613.8 1.07 1.016q.713.35 1.753.35a5 5 0 0 0 .772-.053 3.6 3.6 0 0 0 .667-.163q.316-.11.643-.263l.082-.968a8 8 0 0 1-.93.297 4.4 4.4 0 0 1-1.081.123 3.2 3.2 0 0 1-.866-.105 1.33 1.33 0 0 1-.988-.981 3 3 0 0 1-.097-.512l2.032-.107q.46-.026.9-.158m-3.052-.602c.015-.285.072-.566.168-.834.096-.284.28-.529.526-.701.254-.158.549-.238.848-.227.242-.01.483.04.701.146.164.083.302.21.398.368a.9.9 0 0 1 .128.443.66.66 0 0 1-.087.345.6.6 0 0 1-.293.233 1.7 1.7 0 0 1-.544.111zm8.532-2.74-.164 1.204a3 3 0 0 0-.45.006q-.204.015-.404.064-.188.046-.368.117a2.4 2.4 0 0 0-.356.175 9 9 0 0 0-.258.163v4.344h-1.333v-5.909l1.333-.198v.938l.065-.074q.243-.274.543-.484c.194-.136.41-.237.637-.298.246-.064.502-.08.755-.047"/><path fill="#fff" fill-rule="evenodd" d="M23.913 5C18.289 5 13.7 9.584 13.7 15.2c0 5.606 4.589 10.2 10.213 10.2 5.612 0 10.212-4.594 10.212-10.2 0-5.616-4.6-10.2-10.212-10.2m-8.21 10.2c0 4.528 3.709 8.276 8.21 8.276 4.544 0 8.21-3.748 8.21-8.276 0-4.583-3.665-8.276-8.21-8.276-4.501 0-8.21 3.693-8.21 8.276m6.405.033a1.55 1.55 0 0 1-1.574 1.566 1.57 1.57 0 0 1-1.568-1.566 1.565 1.565 0 0 1 1.568-1.566 1.555 1.555 0 0 1 1.574 1.566m7.142-.726-1.034.824c-.078.055-.078.11-.055.198l.34 1.264c.023.067-.01.077-.054.044l-1.112-.725a.165.165 0 0 0-.198 0l-1.123.737c-.044.032-.077.01-.055-.055l.341-1.264a.17.17 0 0 0-.044-.198l-1.045-.847c-.033-.032-.033-.076.022-.076l1.31-.055a.13.13 0 0 0 .132-.1l.506-1.263c.033-.077.077-.067.1 0l.44 1.22a.18.18 0 0 0 .198.143l1.298.055c.077 0 .088.054.033.098" clip-rule="evenodd"/></svg>
```

Comment:
if you look at the svg preview (-view?), you an see that not all paths have `fill="currentColor"` easily solved by moving the fill rule to the svg-root and removing all fill-rules on paths

Reply:
But only two of the icons had `fill="currentColor"` before, so I just added it to those that did? I think the PR is correct as is 🤔

Reply:
![Screenshot 2024-08-28 at 12 12 27](https://github.com/user-attachments/assets/46c46934-74df-4510-80ba-108656aa903f)
The monochrome logos and icons should only have `fill="currentColor"` which means they will render with only a single color across all paths.